superset/db_engine_specs/mysql.py

```diff
@@ -53,11 +53,16 @@
 CONNECTION_UNKNOWN_DATABASE_REGEX = re.compile("Unknown database '(?P<database>.*?)'")
 
 
-class MySQLEngineSpec(BaseEngineSpec):
+class MySQLEngineSpec(BaseEngineSpec, BasicParametersMixin):
     engine = "mysql"
     engine_name = "MySQL"
     max_column_name_length = 64
 
+    drivername = "mysql"
```

Comment:
I just realized we should use a different driver here:
```suggestion
    drivername = "mysql+mysqlconnector"
```